packages/web/src/app/admin/products/ProductForm.component.tsx

```diff
@@ -0,0 +1,337 @@
+                            <option>yellow</option>
+                            <option>red</option>
+                            <option>green</option>
+                            <option>blue</option>
+                        </select>
+                    </div>
+                    <div>
+                        <label htmlFor="size" className="block text-sm font-medium leading-6 text-gray-900">
+                            Größe
+                        </label>
+                        <select
+                            id="size"
+                            name="size"
+                            value={formData.size}
+                            onChange={handleChange}
+                            className="mt-2 block w-full rounded-md border-0 px-3 py-1.5 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6">
+                            <option>s</option>
+                            <option>m</option>
+                            <option>l</option>
+                            <option>xl</option>
+                            <option>xxl</option>
+                        </select>
+                    </div>
+                </div>
+
+                {/* Material and Producer Inputs */}
+                <div>
+                    <label htmlFor="material" className="block text-sm font-medium leading-6 text-gray-900">
+                        Material
+                    </label>
+                    <input
+                        type="text"
+                        name="material"
+                        id="material"
+                        value={formData.material}
+                        onChange={handleChange}
+                        required
+                        className="mt-2 block w-full rounded-md border-0 px-3 py-1.5 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6"
+                    />
+                </div>
+                <div>
+                    <label htmlFor="producer" className="block text-sm font-medium leading-6 text-gray-900">
+                        Hersteller
+                    </label>
+                    <input
+                        type="text"
+                        name="producer"
+                        id="producer"
+                        value={formData.producer}
+                        onChange={handleChange}
+                        required
+                        className="mt-2 block w-full rounded-md border-0 px-3 py-1.5 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6"
+                    />
+                </div>
+
```

Comment:
Where is it from? The duck origin is missing here. (I'm not sure if It's a country code input like US/GER/etc. or a string input like Germany/USA/Gurkistan)